packages/api/bin/serve.js

```diff
@@ -112,24 +95,6 @@ async function setTableEnvVariables(stackName) {
   };
 }
 
-// check if the tables and Elasticsearch indices exist
```

Comment:
We may need another method to populate ES indices. This, unfortunately, isn't testable on some of our machines due to the other localstack issues on M1 macs.

Reply:
Hmm, I'm curious what you mean by this - looking at the original base.js createTable there doesn't appear to be anything in that method that creates an ES index/modifies es/ etc?